service-pdf/composer.json

```diff
@@ -14,11 +14,11 @@
         "laminas/laminas-filter": "^2.12.0",
         "laminas/laminas-barcode": "^2.11",
         "tecnickcom/tcpdf": "6.7.5",
-        "setasign/fpdi-tcpdf": "^2.0",
         "laminas/laminas-zendframework-bridge": "^1.1.1",
         "laminas/laminas-log": "^2.13",
         "symfony/validator": "^4.2",
-        "egulias/email-validator": "~1.2"
+        "egulias/email-validator": "~1.2",
+        "setasign/fpdi": "2.3.7"
```

Comment:
A problem to be aware of is that this could all appear to work, all tests pass, UAT works, but resulting printed LPA won't scan. So this needs to be tested by scanning by hand LPA(s) generated from this UAT environment. This is why we don't currently have these packages upgraded. It looks like it ought to be a simple fix... but....

Reply:
That's why I pinned it to the version provided by the old repo. The version of `setasign/fpdi` in the lock file is identical to the one we currently use.